ember-toucan-core/src/components/form/controls/input.hbs

```diff
@@ -0,0 +1,8 @@
+<input
+  class="bg-overlay-1 focus:outline-none shadow-focusable-outline focus:shadow-focus-outline block rounded-sm p-1
+    {{if @isDisabled 'text-disabled' 'text-titles-and-attributes'}}"
```

Comment:
I found out yesterday we need a `@hasError` argument so we can conditionally apply the error shadow.  Something  like this should work:

```suggestion
  class="bg-overlay-1 focus:outline-none focus:shadow-focus-outline block rounded-sm p-1
    {{if @isDisabled 'text-disabled' 'text-titles-and-attributes'}}
    {{if @hasError 'shadow-error-outline' 'shadow-focusable-outline'}}"
```

(For reference:  https://github.com/CrowdStrike/ember-toucan-core/pull/52/files#diff-f86c0f8f074d8a714e4e434f71ef3d3ee3db10702a7260cce69f4143ab2b71f7)

Reply:
Interesting...are we always going to conditionally apply via a `@hasError` argument or should it be `this.hasError` to account for if there are arguments passed to `@error`?

Reply:
oh nvm `@error` comes from the InputField component